### ISSUE TITLE
Obtain citation formatted list of DOIs 

### ISSUE THREAD

**Issue by kjgarza** [datacite/lupo#232]
- [x] implementation. 
https://github.com/datacite/lupo/commit/d7cabfcaf94e482779f71e09b6293b06a4c28d29
- [x] review performance with Gbif list: https://api.datacite.org/events?doi=10.15468/dipjcr

We  will use semi “citation format” to show in the citation list. Based o nthe infomration in EventData.

Content Type, Author, YOP, Title, Journal Title, DOI (with http)

**Comment by mfenner** [comment]
Based on today's discussion this is no longer needed.

**Comment by kjgarza** [comment]
- [ ] remove duplicated links if any. The direct and reverse relation could be returned in the events. Implement a filter to avoid double counting.

**Comment by mfenner** [comment]
This is no longer needed.

**Comment by kjgarza** [comment]
I'm not sure if we are referring to the same thing on this ticket. In my mind, this ticket is about formatting the DOI metadata (in whatever serialization one's get it) to the pseudo citation style we defined in the description of this issue. As far as I can see there are 3 serializations we have been using:

A) the /events?include=subj,obj serialization (this was the first implementation we used)
B) the /works serialization (currently I have implemented this ticket using this serialization)
C) the /dois or /events?includes=dois serialization 

If we change to use (C), then we need to redo this ticket.
